lib/src/components/NxGlobalSidebar/types.ts

```diff
@@ -53,3 +53,25 @@ export const nxGlobalSidebarNavigationLinkPropTypes: ValidationMap<NxGlobalSideb
   text: PropTypes.string.isRequired,
   href: PropTypes.string.isRequired
 };
+
+export interface NxGlobalSidebarFooterProps {
+  supportText?: string | null;
+  supportLink?: string | null;
+  releaseText?: string | null;
+  releaseNumber?: string | null;
+  productTagLine?: string | null;
+  showCreatedBy: boolean;
```

Comment:
I feel like this boolean should probably be optional as well, defaulting one way or the other depending on what we think is more common (`true` I would think)

Reply:
> I feel like this boolean should probably be optional as well, defaulting one way or the other depending on what we think is more common (`true` I would think)

Ok: https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/deb6e5507c9d5586545ae5632577305aced2563f